src/main/java/org/junitpioneer/jupiter/SetEnvironmentVariable.java

```diff
@@ -35,9 +35,9 @@
  * uses reflection to change them. This requires that the {@link SecurityManager}
  * allows modifications and can potentially break on different operating systems and
  * Java versions. Be aware that this is a fragile solution and consider finding a
- * better one for your specific situation. If you're running on Java 9 or later, you
- * may have to add {@code --add-opens=java.base/java.util=ALL-UNNAMED} to your test
- * execution to prevent warnings or even errors.</p>
+ * better one for your specific situation. If you're running on Java 9 or later and
+ * are encountering warnings or errors, check
+ * <a href="https://junit-pioneer.org/docs/environment-variables/">the documentation</a>.</p>
```

Comment:
Could we link to the specific section? Would that be better?

Reply:
Will do.